plugins/modules/remote_management/redfish/redfish_config.py

```diff
@@ -100,6 +100,18 @@
     type: bool
     default: false
     version_added: 3.7.0
+  hostinterface_config:
+    required: false
+    description:
+      - setting dict of HostInterface on OOB controller
```

Comment:
```suggestion
      - Setting dict of HostInterface on OOB controller.
```
Descriptions should always have complete sentences.